src/terminal.ts

```diff
@@ -53,8 +62,48 @@ export default class Terminal {
 
 	}
 
+	public runCommandInOutput (args: string[]) {
+		if (this.proc) {
+			window.showInformationMessage('A build is already in progress');
+			return;
+		}
+		if (!this.channel) {
+			this.channel = window.createOutputChannel('Appcelerator');
+		}
+		args.push('--no-prompt');
+		this.channel.clear();
+		this.channel.append(`${this.command} ${args.join(' ')}\n\n`);
+		this.proc = spawn(this.command, args, { shell: true });
+		this.proc.stdout.on('data', data => {
+			ExtensionContainer.context.globalState.update(GlobalState.Running, true);
+			vscode.commands.executeCommand('setContext', 'build:running', true);
```

Comment:
Use the enum here too
```suggestion
			vscode.commands.executeCommand('setContext', GlobalState.Running, true);
```